packages/dapp-high-roller/src/assets/images/dice/Dark/Dice-Dark-01.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@Alonski These SVGs files could be minified. You can use something like [svgo](https://github.com/svg/svgo) to do it.